src/status_im2/contexts/chat/lightbox/bottom_view.cljs

```diff
@@ -50,15 +50,17 @@
 
 (defn bottom-view
   [messages index scroll-index insets animations derived item-width props]
-  (let [text               (get-in (first messages) [:content :text])
-        padding-horizontal (- (/ item-width 2) (/ c/focused-image-size 2))]
+  (let [{:keys [chat-id content]} (first messages)
+        padding-horizontal        (- (/ item-width 2) (/ c/focused-image-size 2))]
     [reanimated/linear-gradient
      {:colors [:black :transparent]
       :start  {:x 0 :y 1}
       :end    {:x 0 :y 0}
       :style  (style/gradient-container insets animations derived)}
-     (when constants/image-description-in-lightbox?
-       [rn/text {:style style/text-style} text])
+     [message-view/render-parsed-text
+      {:content        content
+       :chat-id        chat-id
+       :style-override style/text-style}]
```

Comment:
imo this approach could be improved upon here.
this prop drilling of some static styles and then later using it at boolean to set an override color to some nested text as white.

Can you give some more context to what you are trying to achieve here so we can try to approach this differently?

Reply:
I want to be able to use the same text styles in the text preview part of lightbox.

I thought I can reuse the same text rendering logic there.

Reply:
If we were to create a separate implementation of this to not hardcode the props, We'd still face duplication.